README.md

```diff
@@ -127,3 +127,49 @@ _output:_
 `["a", "b", "c", "d"]`
 
 
+### Pandas dataframe
+
+
+#### describe()
+[Official Docs](https://www.w3schools.com/python/pandas/ref_df_describe.asp#:~:text=The%20describe()%20method%20returns,The%20average%20(mean)%20value.)
+
+
+The describe() its a pandas dataframe and a method that provides descriptive
```

Comment:
Add quotes around `describe()` so that it's well styled. Trying adding. `

Also, update "its" to "is". 

Also, this is not a pandas dataframe. This is a method (or function). So remove the "is a pandas dataframe"